ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -242,10 +259,25 @@ if [ $verify = upper ] ; then
                 echo "done" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
             fi
         fi
-        chmod +x run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
+
+	# Indicate sub-task is completed for restart 
+	echo ">$COMOUTsmall/run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.completed" >> run_${modnam}_valid_at_t${vhour}z_${fhr}_${metplus_job}_g2g.sh
```

Comment:
This line here and other like it are writing a file directly to COMOUT, which we cannot do.

I would suggest writing these completed files somewhere in DATA, and then using `if [ SENDCOM = YES]; then` to copy it to COMOUTsmall.

Reply:
Fixed in the new commit.